lib/uipage/widgets/questions_list.dart

```diff
@@ -1,197 +1,184 @@
 import 'dart:convert';
-
 import 'package:chopper/chopper.dart';
 import 'package:flutter/material.dart';
-import 'package:intl/intl.dart';
-import 'package:built_collection/built_collection.dart';
 import 'package:referease/commonwidget/fancybutton.dart';
 import 'package:referease/data/api_functions/questionnaire/request_questions.dart';
 import 'package:referease/uipage/summary/summarydetail.dart';
 import 'package:referease/uiutility/colors.dart';
 
-import '../../model/question_model.dart';
-
+class QuestionsList extends StatefulWidget {
+  @override
+  _QuestionsList createState() => _QuestionsList();
 
-class QuestionsList extends StatelessWidget{
   final int id;
   final String authors;
   final String title;
   final int questionnaire_type_id;
 
-  QuestionsList({this.id, this.authors, this.title, this.questionnaire_type_id});
+  QuestionsList(
+      {this.id, this.authors, this.title, this.questionnaire_type_id});
+}
+
+class _QuestionsList extends State<QuestionsList> {
+  dynamic questionDetails;
+
+  @override
+  void initState() {
+    super.initState();
+  }
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-           appBar:AppBar(
+      appBar: AppBar(
         brightness: Brightness.light,
```

Comment:
set an elevation to zero on the app bar. If you are using app bar elsewhere please set the elevation to 0